weather.css

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
Please use English in your commit names, tags, etc. Also, ensure that commit messages are more descriptive(sensible), so the person reviewing can understand what changes were made in the commit.

Reply:
Also this file should be called style.css not weather.css since this is one and only global purpose css file